ui/app/components/app/modals/modal.js

```diff
@@ -80,9 +80,9 @@ const accountModalStyle = {
 
 const MODALS = {
   DEPOSIT_ETHER: {
-    contents: [
-      h(DepositEtherModal, {}, []),
-    ],
+    contents: [(
+      <DepositEtherModal key={0} />
```

Comment:
Nit: All of these single-entry arrays could be assigned the component directly instead. This `contents` is just passed as the `children` prop to the `FadeModal` - it's never assumed to be an array.

Reply:
Good point, I'll update these